src/tinybox.vue

```diff
@@ -456,4 +459,4 @@ export default {
   .prev-enter-active {
     transition: opacity 300ms ease, transform 300ms ease;
   }
-</style>
+</style>
```

Comment:
Oh, and please add the line break at the end here 😊

Reply:
🤦 sorry dunno what happened